asserts/toolbelt/main.go

```diff
@@ -1,204 +0,0 @@
-// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
well the other changes break it, and it was never advertized (it's really demoware that so far compiled but what already conceptually broken)